monkey/infection_monkey/payload/ransomware/readme_dropper.py

```diff
@@ -11,4 +10,7 @@ def leave_readme(src: Path, dest: Path):
         return
 
     logger.info(f"Leaving a ransomware README file at {dest}")
-    shutil.copyfile(src, dest)
+    # Opening file to convert line endings
```

Comment:
Let's add a little more detail/context in this comment. Maybe even a link to the github issue.